samza-api/src/main/java/org/apache/samza/container/placement/ContainerPlacementResponseMessage.java

```diff
@@ -42,7 +42,7 @@ public ContainerPlacementResponseMessage(UUID uuid, String deploymentId, String
     this(uuid, deploymentId, processorId, destinationHost, null, statusCode, responseMessage, timestamp);
   }
 
-  static ContainerPlacementResponseMessage fromContainerPlacementRequestMessage(
+  public static ContainerPlacementResponseMessage fromContainerPlacementRequestMessage(
```

Comment:
/**
Helper function to create a {@link ContainerPlacementResponseMessage} using BLA-BLA params of 
{@link ContainerPlacementRequestMessage} and the given additional parameters.
*/

Reply:
Sure adding docs